ibtk/include/ibtk/GeneralOperator.h

```diff
@@ -123,6 +128,16 @@ class GeneralOperator : public virtual SAMRAI::tbox::DescribedClass
      */
     virtual SAMRAI::tbox::Pointer<HierarchyMathOps> getHierarchyMathOps() const;
 
+    /*!
+     *  \brief Set the ProblemSpecification object used by the operator.
+     */
+    virtual void setProblemSpecification(const ProblemSpecification* problem_spec);
```

Comment:
I agree that this should be a feature of derived classes. Furthermore - we need those parameters to actually do anything, right? They should be constructor arguments so we don't have half-constructed classes.